Users/Glob_Patterns.rst

```diff
@@ -180,8 +174,6 @@ brackets. Parentheses that have no match are ignored as well as
     True
```

Comment:
I'd say we can remove L177 too (`fnmatch("abc", "a*c")`) since that is anyway covered with `abbc`.